package.json

```diff
@@ -175,6 +175,7 @@
     "nouislider": "9.2.0",
     "numeral": "2.0.6",
     "openseadragon": "2.4.2",
+    "os-browserify": "0.3.0",
```

Comment:
What is this one?

Reply:
Yeah, recommended in the Webpack 5 migration guide.